dhis-2/dhis-api/src/main/java/org/hisp/dhis/tracker/imports/validation/ValidationCode.java

```diff
@@ -157,6 +157,7 @@ public enum ValidationCode {
   E1312("Referral events need to have both sides of a relationship."),
   E1313(
       "Event {0} of an enrollment does not point to an existing tracked entity. The data in your system might be corrupted"),
+  E1315("Status `{0}` does not allow to define data values."),
```

Comment:
Should we print a static list of event status that do not allow setting data values? As in give the user all the information needed to fix their problem.